src/components/sidebar/NewTxButton/index.tsx

```diff
@@ -14,10 +16,18 @@ const NewTxButton = (): ReactElement => {
   const isSafeOwner = useIsSafeOwner()
   const isWrongChain = useIsWrongChain()
 
+  const onClick = () => {
+    setTxOpen(true)
+
+    if (wallet && !isWrongChain && isSafeOwner) {
+      trackEvent(OVERVIEW_EVENTS.NEW_TRANSACTION)
+    }
```

Comment:
Is this necessary? AFAIK disabled buttons don't trigger onClick. Could be tracked via `<Track>`.

Reply:
I can remove the condition as the button doesn't trigger `onClick` indeed. `<Track>` would track the click though.